src/response.ts

```diff
@@ -46,10 +38,26 @@ const stringifyJsonBody: ResponseTransformer = (res) => {
   return res
 }
 
-function createResponseComposition(
-  overrides: Partial<MockedResponse> = {},
+export const defaultResponse: Omit<MockedResponse, 'headers'> = {
+  status: 200,
+  statusText: 'OK',
+  body: null,
+  delay: 0,
+  once: false,
+}
+
+export type ResponseCompositionOptions<BodyInit> = {
+  defaultTransformers?: ResponseTransformer<BodyInit>[]
+  mockedResponseOverrides?: Partial<MockedResponse>
+}
+
+export function createResponseComposition<BodyInit>(
+  options: ResponseCompositionOptions<BodyInit> = {
+    defaultTransformers: [stringifyJsonBody],
+    mockedResponseOverrides: {},
+  },
```

Comment:
I have added not in the `MockedResponse` type because it was something not problem regarding to the response